### PR TITLE
Fix typo in launch.py

### PR DESCRIPTION
**Pull request by datakurre**

### CODE REVIEW

File: robotlab/src/robotlab/launch.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
 CMD = [
     "call python -m robotlab.labapp"
 ] if WIN else [
-    "pyton -m robotlab.labapp"
+    "python -m robotlab.labapp"
```
Comment (bollwyvl) [inline robotlab/src/robotlab/launch.py:27]: woof. good catch, thanks.

**Comment by bollwyvl** [comment]
Might be in and out for a bit, please feel free to merge if it comes up :green_heart: !

**Comment by bollwyvl** [comment]
Thx!

**Comment by datakurre** [comment]
Confirmed that RobotLab shortcut works now on OSX.

**Comment by bollwyvl** [comment]
Hooray! Shortcuts for everyone! Sorry I closed it too early!

**Pull request merged**